tests/pkg/fixtures/fixtures_queries.go

```diff
@@ -0,0 +1,602 @@
+/*
+ * Copyright 2020 The Compass Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fixtures
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-incubator/compass/tests/pkg"
+	gqlTools "github.com/kyma-incubator/compass/tests/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/pkg/jwtbuilder"
+	"github.com/kyma-incubator/compass/tests/pkg/testctx"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	gcli "github.com/machinebox/graphql"
+
+	"github.com/stretchr/testify/require"
+)
+
+//Application
```

Comment:
Does it make sense to distribute the different types into their own files, e.g. `application_queries.go`? We can skip the `fixtures` prefix since that's the package (and folder) name.

The same can be applied to the `fixtures_requests.go`, and at the end we'll have a bunch of files like:
```
application_queries.go
application_requests.go
api_spec_queries.go
...
``` 

Or even merge the `_queries` and `requests` and leave just:
```
applications.go
runtimes.go
bundles.go
...
```